iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Identifiable.java

```diff
@@ -74,4 +74,18 @@ default Properties getProperties() {
      * Get properties key values.
      */
     Set<String> getPropertyNames();
+
+    /**
+     * Get the fictitious status of the switch
+     */
+    default boolean isFictitious() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
This method could return false by default instead of throwing an exception.
@MioRtia What do you think?

Reply:
Both solutions are good to me!